README.md

```diff
@@ -173,8 +172,7 @@ typst help watch
 ```
 
 If you prefer an integrated IDE-like experience with autocompletion and instant
-preview, you can also check out the [Typst web app][app], which is currently in
-public beta.
+preview, you can also check out [Typst's free web app.][app]
 
 ## Community
 The main place where the community gathers is our [Discord server][discord].
```

Comment:
```suggestion
The main place where the community gathers is our [Discord server.][discord]
```
For consistency with the links above.

Reply:
I'd argue that the stop shouldn't be part of the link

Reply:
While it's a bit weird, to my knowledge, it is a common typographical practice to include punctuation in styled segments (like strong, emph, quotes), so I think this would naturally extend to links as well.

Reply:
However, the remaining README doesn't do this anywhere, so I'll move it out.